manifests/x/XianWei/StarVPN/5.2.5/XianWei.StarVPN.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: XianWei.StarVPN
+PackageVersion: 5.2.5
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Sword6988/StarVPN/releases/download/v5.2.5/StarVPN.exe

```

Comment:
I don't see how to get to this link from the Package URL. Please provide documentation that this is an official URL provided by StarVPN. 

WinGet policies prohibit adding applications re-hosted on personal repositories